__mocks__/fs/promises.cjs

```diff
@@ -0,0 +1,2 @@
+const { fs } = require('memfs')
+module.exports = fs.promises
```

Comment:
does it mean, all tests will have the `node:fs` and `node:fs/promise` mocked? or only when calling `vi.mock('node:fs')` ?

Reply:
AFAIK only when using vi.mock()